code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -218,6 +218,16 @@
 	LoseTarget()
 	..(gibbed)
 
+/mob/living/simple_animal/hostile/proc/summon_backup(distance)
+	do_alert_animation(src)
+	playsound(loc, 'sound/machines/chime.ogg', 50, 1, -1)
+	mob_loop:

```

Comment:
@Thunder12345
Any time you might be using a named loop, there's almost always a better way
Use Razh's suggestion